examples/google-map-with-markers/components/map-markers/map-markers.tsx

```diff
@@ -54,36 +54,37 @@ const MapMarkers: VoidFunctionComponent = (): null => {
 
   const [, setMarkers] = useState<Array<google.maps.Marker>>([]);
 
+  // Add markers to the map
```

Comment:
I cannot comment it here, but for line 51, can we also remove the `VoidFunctionComponent` here in this PR, just saw it.

> 'VoidFunctionComponent' is deprecated.ts(6385)

Reply:
Already done in #100